treeherder/webapp/api/performance_data.py

```diff
@@ -64,7 +64,14 @@ def list(self, request, project):
                 framework__in=frameworks)
 
         interval = request.query_params.get('interval')
```

Comment:
Or maybe you just mean so you have the option to use start_date and end_date, -OR- just the interval?